packages/playground/src/dashboard/components/reserve_action_btn.vue

```diff
@@ -121,7 +126,7 @@ export default {
         }, 20000);
       } catch (e) {
         console.log("Error: ", e);
-        createCustomToast(e as string, ToastType.danger);
+        createCustomToast(normalizeError(e, "Failed to rent node."), ToastType.danger);
```

Comment:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/4d59fbda-034b-4f4c-ab26-70eb08df57c6)

i tried to remove the @checkBalance decorator to get  chain error and its not the wanted error msg, we can make it more clear to the user